v1/gap_appearance.json

```diff
@@ -1,54 +1,1282 @@
 [
-{"category": "000", "name": "Unknown"},
```

Comment:
Drone

Reply:
What's that...? Category 000 would be a drone...?